cypress/integration/add_record_adult_admin.ts

```diff
@@ -0,0 +1,37 @@
+describe('Adult Group', () => {
+  beforeEach(() => {
+    cy.getCookies().should('be.empty');
+    cy.setCookie('hackneyToken', Cypress.env('TEST_KEY_ADULT_GROUP'));
+    cy.getCookie('hackneyToken').should(
+      'have.property',
+      'value',
+      Cypress.env('TEST_KEY_ADULT_GROUP')
+    );
+    cy.visit(Cypress.env('HOST') + '/search');
+  });
+
+  it('Confirm person type is adult', () => {
```

Comment:
This is already covered by https://github.com/LBHackney-IT/lbh-social-care-frontend/blob/main/cypress/integration/search_by_name_adult_admin.ts, not sure of the value it will bring 🤔